config/main.py

```diff
@@ -464,6 +470,28 @@ def portchannel_member(ctx):
 def add_portchannel_member(ctx, portchannel_name, port_name):
     """Add member to port channel"""
     db = ctx.obj['db']
+    port_channel = db.get_entry('PORTCHANNEL', portchannel_name)
+    if get_interface_naming_mode() == "alias":
+        port_name = interface_alias_to_name(port_name)
+        if port_name is None:
+            ctx.fail("'port_name' is None!")
+    
+    if len(port_channel) == 0:
+        ctx.fail("{} doesn't exists".format(portchannel_name))
+    
+    members = port_channel.get('members', [])
+    if port_name in members:
+        if get_interface_naming_mode() == "alias":
+            port_name = interface_name_to_alias(port_name)
+            if port_name is None:
+                ctx.fail("'port_name' is None!")
+            ctx.fail("{} is already a member of {}".format(port_name, portchannel_name))
+        else:
+            ctx.fail("{} is already a member of {}".format(port_name, portchannel_name))
+    
+    members.append(port_name)
+    port_channel['members'] = members
```

Comment:
the attribute 'members' is actually not used any more in the latest code base; i think this is not needed